CONTRIBUTING.md

```diff
@@ -1,11 +1,19 @@
-## How to Contribute
+How to Contribute
+=================
 
 As an open-source project, Chapel welcomes source contributions from
 the community.  Contributors are asked to sign a contributor agreement
 in order to help us maintain Chapel's license terms.  The
-contributor's agreement can be found
-[here](https://github.com/chapel-lang/chapel/tree/master/doc/developer/contributorAgreements/).
+contributor's agreement can be found in [doc/developer/contributorsAgreements/][0].
 
-For more detailed instructions on committing code to Chapel, please
-refer to the contributor's
-[README](https://github.com/chapel-lang/chapel/tree/master/doc/developer/bestPractices/ContributorInfo.txt).
+For more detailed instructions on committing code to Chapel, please refer to
+[ContributorInfo.txt][1]. The high level steps are:
```

Comment:
I'd suggest a slight reorganization here for clarity.  Essentially:

The high-level steps for contributing code to Chapel are:

1 ...
2 ...
3 ...
4 ...
5 ...

For more detailed instructions, please refer to ...